### DISCUSSION TITLE
Welcome to rustwiki.org Discussions!

### DISCUSSION THREAD

**Discussion by aaranxu** [rust-lang-cn/rustwiki.org#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ikilobyte** [comment]
有没有微信群

**Reply by uyplayer** [reply]
同问

**Comment by AirY-Foo** [comment]
learn Rust

**Reply by WeianH** [reply]
fn main() {
    let s1 = String::from("hello");

    let (s1, len) = calculate_length(s1);

    println!("The length of '{}' is {}.", s1, len);
}

fn calculate_length(s: String) -> (String, usize) {
    let length = s.len(); // len() 返回字符串的长度

    (s, length)
}

這樣運作也正常 我不知道原本範例 s2 的用意
